dist/index.d.ts

```diff
@@ -1,4 +1,4 @@
-export interface FunctionParams {
+interface FunctionParams {
```

Comment:
Lets remove `dist/` from git, since it's being generated.  We'll publish it to npm, but we won't keep it in git.

```sh
git rm -fr dist
```

Now add `dist` to `.gitignore` so it doesn't include it